### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by iDVB** [gorangajic/react-icons#110]
Just noticing that currently react-icons does not support passing down the `title` and `description` for the icons. Any chance this can be added? Would you accept a PR?
https://css-tricks.com/accessible-svgs/#article-header-id-6

**Comment by silvenon** [comment]
Yeah, this would be awesome.

**Comment by mryechkin** [comment]
Agreed, it would be great to see this added.

The project I'm working on now requires AODA compliance, and this feature is one thing we're having some trouble with for actionable icons. Using a separate label with aria-hidden="true" doesn't work as nicely with iOS voice assistance

**Comment by iDVB** [comment]
I've got this ready to PR, other then I'm trying to figure out the proper unit tests to write for it.
@gorangajic any suggestions on this PR? https://github.com/gorangajic/react-icon-base/pull/25

**Comment by nscarcella** [comment]
Any chance this will get implemented in the near future?

**Comment by kamijin-fanta** [comment]
@silvenon It may be resolved by using react-icons@3.0.0 or higher.

```
<FaFolder aria-labelledby="discribe" />
```

**Comment by Slapbox** [comment]
Should we assume this isn't a planned feature?

**Comment by paustria** [comment]
Please support this feature. I have to move away from `react-icons` because `title` doesn't support and the browser does not read icon.

**Comment by silvenon** [comment]
For those looking to migrate away, you can fairly easily roll your own icon system with `<symbol>`, copying code from [IcoMoon](https://icomoon.io/): https://gist.github.com/silvenon/47857d019ed1fd0c032471604d0c0f53
